### PR TITLE
leading slash handling fixes

### PR DESCRIPTION
**Pull request by marsaud**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,10 @@
     "clarify": "^1.0.5",
     "dependency-check": "^2.5.1",
     "ghooks": "^1.2.1",
+    "lodash.filter": "^4.0.1",
+    "lodash.map": "^4.0.1",
+    "lodash.trim": "^4.0.1",
+    "lodash.trimstart": "^4.4.0",
```
Comment (julien-f) [inline package.json]: I would prefer using directly the `lodash` package as it simplifies code dedupe and packages maintenance.

It's still possible to import only the needed methods: `import forEach from 'lodash.forEach`' (and it's a good idea when compiling code for the web).

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+import trimStart from 'lodash.trimstart'
 
 const sanitizePath = (...paths) => filter(map(paths, s => s && filter(map(s.split('/'), trim)).join('/'))).join('/')
 
 export const parse = (remote) => {
   const [type, rest] = remote.url.split('://')
   if (type === 'file') {
     remote.type = 'local'
-    remote.path = `/${rest}` // FIXME the heading slash has been forgotten on client side first implementation
+    remote.path = `/${trimStart(rest, '/')}` // the heading slash has been forgotten on client side first implementation
```
Comment (julien-f) [inline src/index.js]: _heading_ → _leading_.

**Comment by julien-f** [comment]
Please, add the related tests.

File: package.json (hunk 1 above)
Comment (marsaud) [inline package.json]: lodash/forEach, right ?

Comment (julien-f) [inline package.json]: Right :)

**Comment by julien-f** [comment]
Thanks!

**Pull request merged**